packages/gasket-plugin-https-proxy/lib/actions.js

```diff
@@ -1,8 +1,19 @@
 import proxy from 'http-proxy';
 
+/**
+ * Gasket action: prepareProxyServer
+ * @param {import('@gasket/core').Gasket} gasket Gasket instance
+ * @returns {Promise<void>} promise
+ * @public
+ */
+export async function prepareProxyServer(gasket) {
+  await gasket.isReady;
+  await gasket.exec('preboot');
+}
+
 /** @type {import('@gasket/core').ActionHandler<'startProxyServer'>} */
 export async function startProxyServer(gasket) {
-  await gasket.isReady;
+  await gasket.actions.prepareProxyServer();
```

Comment:
I don't think we would ever want users execute the `preboot` outside the context of `startProxyServer`, would we? In which case the added `prepareProxyServer` action seem unnecessary. Instead, just kick off the preboot lifecycle when starting.

Also, having 2 different plugins capable of executing the same lifecycles seems dangerous. In theory, both the http and http-proxy plugins could be in used with the same app. Let's distinguish the lifecycles by name, which will also plugins differentiate what they're hooking as needed. Perhaps `preboot` for https plugin, and `prebootHttpsProxy` here?

```suggestion
  await gasket.isReady;
  await gasket.exec('prebootHttpsProxy');
```